src/ExpressServer.js

```diff
@@ -1,17 +1,19 @@
-const express = require('express');
-const cors = require('cors');
-const boredApiRouteHandler=require('./Routes/ExperimentOne/BoredApiCaller')
+const express = require("express");
+const cors = require("cors");
+const boredApiRouteHandler = require("./Routes/ExperimentOne/BoredApiCaller");
+const tmdbApiRouteHandler = require("./Routes/ExperimentTwo/TMDBApiCaller");
 
 const app = express();
 
 //MiddleWare Setup
-app.use(cors());
+app.use(cors()); // Is this required to get the Keys etc
```

Comment:
not sure what this comment is meant to be saying.. but the CORS middleware is used to set some CORS headers in the response, which allows your ui to talk to a backend on a different origin.

Reply:
Yeah realised it wasn't too clear. I wanted to know is this step performed to get api-key or set up some configurations before the server actually starts working